lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -50,10 +50,9 @@ const GatherRunner = {
 };
 
 /**
- * @param {RecursivePartial<LH.Config.Json>} json
+ * @param {LH.Config.Json} json
```

Comment:
I wasn't able to make `RecursivePartial<LH.Config.Json>` work, but `LH.Config.Json` is so close to fully optional that the only thing needed to be used directly was sprinkling in a few `passName`s in the test configs below. Since it's such a small amount of extra setup and we did make `passName` required on purpose, it seemed like a good tradeoff to make.